src/main/java/core/basesyntax/strategy/impl/ReturnOperation.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.OperationHandler;
+
+public class ReturnOperation implements OperationHandler {
+    public void handleOperation(FruitTransaction fruitTransaction, Storage storage) {
```

Comment:
According to the encapsulation principle, it's a good practice to restrict the access to class members. The `handleOperation` method should be marked with an access modifier, most likely `public` if it's intended to be used outside the package.